### ISSUE TITLE
Handle large datasets when populating combo boxes.

### ISSUE THREAD

**Issue by NigelPalmer** [MauroDataMapper/mdm-explorer#166]

**Comment by jamesrwelch** [comment]
@NigelPalmer Good point.  I think enumerations will typically be limited to 20 or so items - certainly we limit at 20 we when do automatic scraping.  We will need a different component to explore terminology types, and so we could repurpose that for very large enumerations.   But fine to start by assuming they'll all be less than 20

**Comment by pjmonks** [comment]
@jamesrwelch Is it possible to provide a data model where there would be a lot of enumerations or terms to test with? We would need to realistically see what such a model looks like.

**Comment by pjmonks** [comment]
In #169 I'm building a control called `mdm-autocomplete-select` to handle referencing Terminologies and large lists of Terms. This control could be used for this case also. For Term search, the backend is queried when typing and pages 20 results at a time. In the case or large enumeration lists this could all be done locally but still only search for 20 at a time.
